code/modules/mob/living/living.dm

```diff
@@ -628,7 +628,7 @@
 	if(layer == initial(layer)) //to avoid things like hiding larvas.
 		layer = LYING_MOB_LAYER //so mob lying always appear behind standing mobs
 	add_traits(list(TRAIT_UI_BLOCKED, TRAIT_PULL_BLOCKED), LYING_DOWN_TRAIT)
-	set_density(FALSE) // We lose density and stop bumping passable dense things.
+	update_density()
```

Comment:
why do this manually exactly? traits are right there

Reply:
I kinda just figured it would make sense to let being prone continue to effect density without a trait? Override me if you think that's dumb.

Reply:
I think it's a bit silly when you already have a system built out to handle it